src/main/java/de/internetsicherheit/brl/bloxberg/cache/Launcher.java

```diff
@@ -7,7 +7,8 @@
 public class Launcher {
 
     public static void main(String[] args) throws IOException {
-        String[] parsedArgs = parseArgs(args);
+        ArgParser parser = new ArgParser();
+        String[] parsedArgs = parser.parseArgs(args);
         if (parsedArgs[4].equals("gui")) {
```

Comment:
Once refactored, this code would become something like:
```Java
if (parsedArgs.isGui()) {
   ...
}

```

As you can see, this provides much better encapsulation.